### DISCUSSION TITLE
"Forbidden download" on Android

### DISCUSSION THREAD

**Discussion by GitCat864** [ppy/osu#13399] [Q&A]
When I try to download Osu!lazer an error (Forbidden download)  appears and won't let me download the game

**Comment by frenzibyte** [comment]
You haven't answered the question raised in https://github.com/ppy/osu/issues/13392#issuecomment-857116194, what do you mean by "forbidden download".

**Reply by GitCat864** [reply]
a ._. I'm distracted
When I try to download the game, the download is canceled and an error occurs (forbidden download) 
Bad eng cuz I'm mexican

**Reply by gagahpangeran** [reply]
Did the error occurs when you download in browser from github release page or when updating inside the game?

**Reply by frenzibyte** [reply]
Screenshot the error so we can understand which part you're having issues with.

**Reply by GitCat864** [reply]
![IMG_20210609_130224.jpg](https://user-images.githubusercontent.com/85586674/121413620-19462d80-c923-11eb-9db3-e42df2b2710b.jpg)
The download just doesn't finish

**Reply by smoogipoo** [reply]
This is going to be an issue with your device or your internet connection (perhaps IP blocked). We can't help.

**Reply by GitCat864** [reply]
> Did the error occurs when you download in browser from github release page or when updating inside the game?

    Maybe it's work if I change my DNS
